packages/components/src/form-token-field/style.scss

```diff
@@ -12,6 +12,13 @@
 
 	&.is-active {
 		@include input-style__focus();
+		border-color: $components-color-accent;
+		box-shadow: 0 0 0 ($border-width-focus - $border-width) $components-color-accent;
+	}
+
+	&:focus {
+		border-color: $components-color-accent;
+		box-shadow: 0 0 0 ($border-width-focus - $border-width) $components-color-accent;
```

Comment:
`border-color` and `box-shadow` were previously defined by `@wordpress/base-styles` for both `.is-active` and `:focus`. For now, overrides have been added to the component styles to allow theming to take effect.